server/server/src/lib.rs

```diff
@@ -88,7 +88,7 @@ async fn run_stage0(
     {
         service_provider
             .app_data_service
-            .set_hardware_id(uuid().to_ascii_uppercase())?;
+            .set_hardware_id(machine_uid::get().unwrap())?;
```

Comment:
You could use .except("Failed to generate machine uid") rather then just panic... or use a default dir? If start with the default config (no local.yaml) its tricky to find out how to get into bootstrap mode...